### PR TITLE
Fix notebooks

### PR DESCRIPTION
**Pull request by sharanry**
As discussed in #1812

### CODE REVIEW

**Comment by menshikh-iv** [comment]
@sharanry what's order of merge for this PR (should we merge this before #1812 or otherwise)?

File: docs/notebooks/Poincare Evaluation.ipynb [hunk 1]
```diff
@@ -933,106 +933,6 @@
        "        <th>100</th>\n",
        "        <th>200</th>\n",
        "    </tr>\n",
-       "    <tr>\n",
```
Comment (menshikh-iv) [inline docs/notebooks/Poincare Evaluation.ipynb]: Why this removed?

File: docs/notebooks/Poincare Evaluation.ipynb [hunk 2]
```diff
@@ -1318,18 +1218,18 @@
...
     {
      "data": {
       "text/html": [
        "<table>\n",
-       "    <tr>\n",
-       "        <th colspan=\"1\" style=\"text-align:left\">WordNet Link Prediction</th>\n",
-       "        <th colspan=\"1\"></th>\n",
-       "        <th colspan=\"6\" style=\"text-align:center\"> Dimensions</th>\n",
-       "    </tr>\n",
+       "        <tr>\n",
```
Comment (menshikh-iv) [inline docs/notebooks/Poincare Evaluation.ipynb]: Why diff here?

File: docs/notebooks/Poincare Evaluation.ipynb [hunk 3]
```diff
@@ -1340,106 +1240,6 @@
        "        <th>100</th>\n",
        "        <th>200</th>\n",
        "    </tr>\n",
-       "    <tr>\n",
```
Comment (menshikh-iv) [inline docs/notebooks/Poincare Evaluation.ipynb]: Why this removed (and exactly same question for similar blocks after).

File: gensim/scripts/word2vec2tensor.py [hunk 4]
```diff
@@ -57,7 +57,7 @@ def word2vec2tensor(word2vec_model_path, tensor_filename, binary=False):
     outfiletsvmeta = tensor_filename + '_metadata.tsv'
 
     with open(outfiletsv, 'w+') as file_vector:
-        with open(outfiletsvmeta, 'w+') as file_metadata:
+        with open(outfiletsvmeta, 'wb+') as file_metadata:
```
Comment (menshikh-iv) [inline gensim/scripts/word2vec2tensor.py]: Why did this need? python3.5 problem?

File: docs/notebooks/Poincare Evaluation.ipynb (hunk 2 above)
Comment (sharanry) [reply docs/notebooks/Poincare Evaluation.ipynb]: This wasn't done manually, when i ran the notebook in the docker container the output might have been rendered differently.

File: docs/notebooks/Poincare Evaluation.ipynb (hunk 3 above)
Comment (sharanry) [reply docs/notebooks/Poincare Evaluation.ipynb]: Again this wasnt removed intentionally... the model was taking too long to run on my local system. I ended up just test running some of these cells(they are all working) and interrupting between. So the output is not present.

File: gensim/scripts/word2vec2tensor.py (hunk 4 above)
Comment (sharanry) [reply gensim/scripts/word2vec2tensor.py]: `%run ../../gensim/scripts/word2vec2tensor.py -i doc_tensor.w2v -o movie_plot` This line of code in `Tensorboard_visualizations.ipynb` fails without this change

File: docs/notebooks/Poincare Evaluation.ipynb (hunk 1 above)
Comment (sharanry) [reply docs/notebooks/Poincare Evaluation.ipynb]: Again I wasn't able to complete the execution of this cell as it was taking too long

Comment (menshikh-iv) [reply docs/notebooks/Poincare Evaluation.ipynb]: This isn't good, this information **must be** in this notebook.

File: docs/notebooks/Poincare Evaluation.ipynb (hunk 3 above)
Comment (menshikh-iv) [reply docs/notebooks/Poincare Evaluation.ipynb]: same comment as before https://github.com/RaRe-Technologies/gensim/pull/1893#discussion_r167535243

File: gensim/scripts/word2vec2tensor.py (hunk 4 above)
Comment (menshikh-iv) [reply gensim/scripts/word2vec2tensor.py]: Can you check this script with other python versions (`2.7` or `3.6`)? This is really suspicious.

File: docs/notebooks/Poincare Evaluation.ipynb (hunk 1 above)
Comment (sharanry) [reply docs/notebooks/Poincare Evaluation.ipynb]: Okay i will try running them again

File: gensim/scripts/word2vec2tensor.py (hunk 4 above)
Comment (sharanry) [reply gensim/scripts/word2vec2tensor.py]: Okay

**Comment by sharanry** [comment]
> @sharanry what's order of merge for this PR (should we merge this before #1812 or otherwise)?
We need to merge this after that one, we may have to resolve one or two merge conflicts while merging this

**Comment by menshikh-iv** [comment]
Please move current commits to #1812 (see https://github.com/RaRe-Technologies/gensim/pull/1812#issuecomment-366585419)

**Pull request closed without merging**